src/hypertrace/agent/config/test_config.py

```diff
@@ -121,3 +124,10 @@ def unset_env_variables():
     for key in os.environ:
         if key[0:3] == "HT_":
             del os.environ[key]
+
+
+def test_telemetry_constants():
```

Comment:
I am not sure about this test, do we need to test constants?

Reply:
I added this yesterday to validate my local change.  I agree and will remove this

Reply:
It's gone.